packages/lesswrong/components/posts/EAPostsItem.tsx

```diff
@@ -412,11 +428,11 @@ const EAPostsItem = ({
                 hasImage && classes.cardTextWithImage,
                 !hasImage && classes.cardTextNoImage,
               )}>
-                {post.contents?.plaintextDescription}
+                {body}
               </div>
               {hasImage &&
                 <img
-                  src={post.socialPreviewData.imageUrl}
+                  src={formatImageUrl(post.socialPreviewData.imageUrl)}
```

Comment:
How was this ever working

Reply:
`imageUrl` is a full image url - it just doesn't have the exact cloudinary parameters that we want, so `formatImageUrl` jerry-rigs them on.